pkg/v1/tkg/client/common.go

```diff
@@ -81,6 +82,65 @@ func checkDockerDaemonIsRunning() (bool, error) {
 	return true, nil
 }
 
+func checkDockerResourceCpu() (int, error) {
```

Comment:
IMO `checkDockerResourceCpu()` and `checkDockerResourceMemory()` can be combined into a single validation function like `checkDockerResource(resourceType string)` and that function can be called from validateDockerPrerequisites as follows 

```
func (c *TkgClient) ValidatePrerequisites {
  ...
  if dockerResourceCpus, err = checkDockerResource("NCPU"); err != nil {
   ... 
 }
 if  dockerResourceTotalMemory, err = checkDockerResource("MemTotal"); err != nil {
  ...
 }
}
```

Please let me know what you think